serverless/aws/lambda-cats-effect/src/main/scala/sttp/tapir/serverless/aws/lambda/runtime/AwsLambdaRuntimeInvocation.scala

```diff
@@ -56,7 +58,7 @@ object AwsLambdaRuntimeInvocation extends StrictLogging {
 
     val routeRequest: (RequestEvent, AwsRequest) => F[Either[Throwable, AwsResponse]] = (event, request) =>
       route(request).map(_.asRight[Throwable]).handleError { case e =>
-        logger.error(s"Failed to process request event ${event.requestId}", e)
+        logger.error("Failed to process request event {}", event.requestId: Any, e: Any)
```

Comment:
Scala 2.12 won't compile this without ascription to `Any` as the method signature is ambiguous.

Reply:
this happens - maybe add a comment in the code, explaining why this is needed?